src/clj_kondo/core.clj

```diff
@@ -21,8 +21,11 @@
 (defn print!
   "Prints the result from `run!` to `*out*`. Returns `nil`. Alpha,
   subject to change."
-  [{:keys [:config :findings :summary :analysis]}]
+  [{:keys [:config :findings :summary :analysis :report-level]}]
   (let [output-cfg (:output config)
+        report-level (keyword report-level)
+        report-level? (set (drop-while #(not= report-level %)
```

Comment:
I think when report-level isn't set, then we can just make this function `(constantly? true)` to be on the safe side when people have added custom levels

Reply:
Also `(keyword nil)` should probably be prevented.